api/net/Keycloak/CssHelper.cs

```diff
@@ -189,19 +189,20 @@ private async Task AddOrUpdateUserAsync(string username, Entities.User? user, TN
                 user.EmailVerified = principal.GetEmailVerified() ?? false;
 
                 // Apply the preapproved roles to the user.
-                var roles = await UpdateUserRolesAsync(key, user.Roles.Split(",").Select(r => r[1..^1]).ToArray());
```

Comment:
I don't understand this logic here.  Why would we remove the first character from every role?